master/internal/db/postgres_checkpoints.go

```diff
@@ -111,3 +117,60 @@ func (db *PgDB) GroupCheckpointUUIDsByExperimentID(checkpoints []uuid.UUID) (
 
 	return groupeIDcUUIDS, nil
 }
+
+// UpdateCheckpointSize updates checkpoint size and count to experiment and trial.
+func UpdateCheckpointSize(checkpoints []uuid.UUID) error {
+	trialID := Bun().NewSelect().Table("checkpoints_view").
```

Comment:
I'd say in retrospect this function would've looked more readable as a raw sql snippet.